zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/BatchOperationControllerTest.java

```diff
@@ -126,6 +127,49 @@ void shouldReturnFailedBatchOperation() {
             JsonCompareMode.STRICT);
   }
 
+  @Test
+  void shouldReturnFailedBatchOperation() {
+    final var batchOperationKey = "1";
+    final var batchOperationEntity =
+        getFailedBatchOperationEntity(batchOperationKey, BatchOperationState.FAILED);
+
+    when(batchOperationServices.getById(batchOperationKey)).thenReturn(batchOperationEntity);
+
+    webClient
+        .get()
+        .uri("/v2/batch-operations/{batchOperationKey}", batchOperationKey)
+        .accept(MediaType.APPLICATION_JSON)
+        .exchange()
+        .expectStatus()
+        .isOk()
+        .expectBody()
+        .json(
+            """
+          {
+              "batchOperationKey":"1",
+              "state":"FAILED",
+              "batchOperationType":"CANCEL_PROCESS_INSTANCE",
+              "startDate":"2025-03-18T10:57:44.000+01:00",
+              "endDate":"2025-03-18T10:57:45.000+01:00",
+              "operationsTotalCount":10,
+              "operationsFailedCount":0,
+              "operationsCompletedCount":10,
```

Comment:
💭 Semantically, does this make sense? We don't actually know how many failed or completed, right? The partitions might have failed halfway through, before any operations were attempted, or after all were completed.